dev/02.安装与配置/02.install-on-k8s.md

```diff
@@ -10,7 +10,7 @@ permalink: /dev/install/install-on-k8s/
 
 ## 步骤 1：准备 Kubernetes 集群
 
-- Kubernetes  <Badge text="v1.16.0" /> 及以上版本 <br>
+- Kubernetes 集群版本：<Badge text="v1.16" /> ~ <Badge text="v1.21" /><br>
```

Comment:
Since the `Ingress` resource was removed during the installation phase, starting from `Zadig 1.8.0`, we can support K8s up to `1.22 `

Reply:
has made a deal offline: announce 1.21 for now.